docs/tutorials/jokes.md

```diff
@@ -4631,7 +4631,7 @@ Awesome! We're ready to handle errors and it didn't complicate our happy path on
 
 Oh, and don't you love how just like with the `ErrorBoundary`, it's all contextual? So the rest of the app continues to function just as well. Another point for user experience 💪
 
-You know what, while we're adding catch boundaries. Why don't we improve the `app/routes/jokes/$jokeId.tsx` route a bit by allowing users to delete the joke if they own it. If they don't, we can give them a 401 error in the catch boundary.
+You know what, while we're adding catch boundaries. Why don't we improve the `app/routes/jokes/$jokeId.tsx` route a bit by allowing users to delete the joke if they own it. If they don't, we can give them a 403 error in the catch boundary. Unlike the 401 when the user needs to log in first, the 403 error means that although the user has been logged in, what he just attempt is forbidden (delete a joke that is not his).
```

Comment:
Thank you for this PR

I wouldn't mind changing 401 to 403 here, but no need to overwhelm the user with details in this tutorial.
Can you remove the added sentence ? 🙏🏼

Reply:
The code sample below that section needs to be updated as well since it's handling the 401

Reply:
Done